src/user/email.js

```diff
@@ -82,7 +83,7 @@ UserEmail.sendValidationEmail = async function (uid, options) {
 	if (plugins.hooks.hasListeners('action:user.verify')) {
 		plugins.hooks.fire('action:user.verify', { uid: uid, data: data });
 	} else {
-		await emailer.send(data.template, uid, data);
+		await emailer.send(data.template, uid, data).catch(err => winston.error('[emailer.send] ' + err.stack));
```

Comment:
This is one case where we might actually want to propagate the error. If a user clicks the "resend" button, we should tell them if the email send failed.